commitlint.config.ts

```diff
@@ -186,6 +194,25 @@ module.exports = {
                     ];
                 },
 
+                'proper-issue-refs': ({raw}: {raw:any}) => {
+                    let offence = false;
+
+                    let rawStr = convertAnyToString(raw, "raw").trim();
+                    let lineBreakIndex = rawStr.indexOf('\n')
+                    
+                    if (lineBreakIndex >= 0){
+                        let bodyStr = rawStr.substring(lineBreakIndex, )
```

Comment:
@realmarv weird way to call the function, I'm surprised it works! let's remove the comma

Reply:
Oh, I forgot to add rawStr.length after the comma. I'm surprised too.

Reply:
Done